person.rb

```diff
@@ -53,10 +53,3 @@ def correct_name
     @nameable.correct_name[0...10]
   end
 end
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227509610-2ffb668e-45eb-48ce-9473-ce878229818b.png)

**[OPTIONAL]**

- [ ] It would be great if you add your name, current year, and app type in your license file to make your project licensed :blush: and you I hope you know the importance of a license in a project because a license tells that this project is legally yours and you are the one who made it :+1:

Reply:
all these changes are optional, so i will implement the at the end of the project, in order to get the right information